app/views/nodes/show.html.erb

```diff
@@ -9,6 +9,12 @@
   <h2><%= @node.name %></h2>
 <% end %>
 
+<% @node.attributes.slice("description", "cost_code").map do |attribute, name| %>
+  <% if name.present? %>
+    <p><b><%= attribute.gsub('_', ' ').capitalize %>:</b> <%= name %></p>
+  <% end %>
+<% end %>
+
```

Comment:
Better use the internationalization variables to get the text for description and cost_code. Take a look at http://guides.rubyonrails.org/i18n.html